AppCenterAnalytics/AppCenterAnalytics/TransmissionTarget/MSAnalyticsAuthenticationProvider.h

```diff
@@ -0,0 +1,70 @@
+#import <Foundation/Foundation.h>
+
+@protocol MSAnalyticsAuthenticationProviderDelegate;
+@class MSAnalyticsAuthenticationResult;
+
+/**
+ *  Different authentication types, e.g. MSA Compact, MSA Delegate, AAD,... .
```

Comment:
I see an extra blank character for this comment before D. Compared to other comments.